proof/refine/Move_R.thy

```diff
@@ -430,4 +430,13 @@ lemma schedule_sched_act_rct[wp]:
 
 (* END: scheduler_action lemmas needed in Refine.thy *)
 
+lemma valid_BlockedOnReceive[simp]:
+  "valid_tcb_state (Structures_A.thread_state.BlockedOnReceive epptr replyOpt payload)
+   = (ep_at epptr and valid_bound_reply replyOpt)"
+  by (rule ext, clarsimp simp: valid_tcb_state_def split: option.splits)
+
+lemma case_option_ext:
```

Comment:
Move?

Reply:
I'm just not sure where to move it. Do you have any suggestions? Otherwise I could just add `FIXME RT: move to Lib?` and leave it here.

Reply:
Nah, no suggestions. Your FIXME sounds good